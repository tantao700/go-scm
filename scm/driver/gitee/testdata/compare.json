{
    "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/compare/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e...7fd1a60b01f91b314f59955a4e4d4e80d8edf11d",
    "html_url": "https://github.com/tantao700/hello-world/compare/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e...7fd1a60b01f91b314f59955a4e4d4e80d8edf11d",
    "permalink_url": "https://github.com/tantao700/hello-world/compare/octocat:553c207...octocat:7fd1a60",
    "diff_url": "https://github.com/tantao700/hello-world/compare/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e...7fd1a60b01f91b314f59955a4e4d4e80d8edf11d.diff",
    "patch_url": "https://github.com/tantao700/hello-world/compare/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e...7fd1a60b01f91b314f59955a4e4d4e80d8edf11d.patch",
    "base_commit": {
        "sha": "553c2077f0edc3d5dc5d17262f6aa498e69d6f8e",
        "node_id": "MDY6Q29tbWl0MTI5NjI2OTo1NTNjMjA3N2YwZWRjM2Q1ZGM1ZDE3MjYyZjZhYTQ5OGU2OWQ2Zjhl",
        "commit": {
            "author": {
                "name": "cameronmcefee",
                "email": "cameron@github.com",
                "date": "2011-01-26T19:06:08Z"
            },
            "committer": {
                "name": "cameronmcefee",
                "email": "cameron@github.com",
                "date": "2011-01-26T19:06:08Z"
            },
            "message": "first commit",
            "tree": {
                "sha": "fcf4a9bba6857422971d67147517eb5edfdbf48d",
                "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/git/trees/fcf4a9bba6857422971d67147517eb5edfdbf48d"
            },
            "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/git/commits/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e",
            "comment_count": 51,
            "verification": {
                "verified": false,
                "reason": "unsigned",
                "signature": null,
                "payload": null
            }
        },
        "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/commits/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e",
        "html_url": "https://github.com/tantao700/hello-world/commit/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e",
        "comments_url": "https://gitee.com/api/v5/repos/tantao700/hello-world/commits/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e/comments",
        "author": null,
        "committer": null,
        "parents": []
    },
    "merge_base_commit": {
        "sha": "553c2077f0edc3d5dc5d17262f6aa498e69d6f8e",
        "node_id": "MDY6Q29tbWl0MTI5NjI2OTo1NTNjMjA3N2YwZWRjM2Q1ZGM1ZDE3MjYyZjZhYTQ5OGU2OWQ2Zjhl",
        "commit": {
            "author": {
                "name": "cameronmcefee",
                "email": "cameron@github.com",
                "date": "2011-01-26T19:06:08Z"
            },
            "committer": {
                "name": "cameronmcefee",
                "email": "cameron@github.com",
                "date": "2011-01-26T19:06:08Z"
            },
            "message": "first commit",
            "tree": {
                "sha": "fcf4a9bba6857422971d67147517eb5edfdbf48d",
                "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/git/trees/fcf4a9bba6857422971d67147517eb5edfdbf48d"
            },
            "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/git/commits/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e",
            "comment_count": 51,
            "verification": {
                "verified": false,
                "reason": "unsigned",
                "signature": null,
                "payload": null
            }
        },
        "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/commits/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e",
        "html_url": "https://github.com/tantao700/hello-world/commit/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e",
        "comments_url": "https://gitee.com/api/v5/repos/tantao700/hello-world/commits/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e/comments",
        "author": null,
        "committer": null,
        "parents": []
    },
    "status": "ahead",
    "ahead_by": 2,
    "behind_by": 0,
    "total_commits": 2,
    "commits": [{
            "sha": "762941318ee16e59dabbacb1b4049eec22f0d303",
            "node_id": "MDY6Q29tbWl0MTI5NjI2OTo3NjI5NDEzMThlZTE2ZTU5ZGFiYmFjYjFiNDA0OWVlYzIyZjBkMzAz",
            "commit": {
                "author": {
                    "name": "Johnneylee Jack Rollins",
                    "email": "johnneylee.rollins@gmail.com",
                    "date": "2011-09-14T04:42:41Z"
                },
                "committer": {
                    "name": "Johnneylee Jack Rollins",
                    "email": "johnneylee.rollins@gmail.com",
                    "date": "2011-09-14T04:42:41Z"
                },
                "message": "New line at end of file. --Signed off by Spaceghost",
                "tree": {
                    "sha": "b4eecafa9be2f2006ce1b709d6857b07069b4608",
                    "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/git/trees/b4eecafa9be2f2006ce1b709d6857b07069b4608"
                },
                "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/git/commits/762941318ee16e59dabbacb1b4049eec22f0d303",
                "comment_count": 213,
                "verification": {
                    "verified": false,
                    "reason": "unsigned",
                    "signature": null,
                    "payload": null
                }
            },
            "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/commits/762941318ee16e59dabbacb1b4049eec22f0d303",
            "html_url": "https://github.com/tantao700/hello-world/commit/762941318ee16e59dabbacb1b4049eec22f0d303",
            "comments_url": "https://gitee.com/api/v5/repos/tantao700/hello-world/commits/762941318ee16e59dabbacb1b4049eec22f0d303/comments",
            "author": {
                "login": "Spaceghost",
                "id": 251370,
                "node_id": "MDQ6VXNlcjI1MTM3MA==",
                "avatar_url": "https://avatars2.githubusercontent.com/u/251370?v=4",
                "gravatar_id": "",
                "url": "https://gitee.com/api/v5/users/Spaceghost",
                "html_url": "https://github.com/Spaceghost",
                "followers_url": "https://gitee.com/api/v5/users/Spaceghost/followers",
                "following_url": "https://gitee.com/api/v5/users/Spaceghost/following{/other_user}",
                "gists_url": "https://gitee.com/api/v5/users/Spaceghost/gists{/gist_id}",
                "starred_url": "https://gitee.com/api/v5/users/Spaceghost/starred{/owner}{/repo}",
                "subscriptions_url": "https://gitee.com/api/v5/users/Spaceghost/subscriptions",
                "organizations_url": "https://gitee.com/api/v5/users/Spaceghost/orgs",
                "repos_url": "https://gitee.com/api/v5/users/Spaceghost/repos",
                "events_url": "https://gitee.com/api/v5/users/Spaceghost/events{/privacy}",
                "received_events_url": "https://gitee.com/api/v5/users/Spaceghost/received_events",
                "type": "User",
                "site_admin": false
            },
            "committer": {
                "login": "Spaceghost",
                "id": 251370,
                "node_id": "MDQ6VXNlcjI1MTM3MA==",
                "avatar_url": "https://avatars2.githubusercontent.com/u/251370?v=4",
                "gravatar_id": "",
                "url": "https://gitee.com/api/v5/users/Spaceghost",
                "html_url": "https://github.com/Spaceghost",
                "followers_url": "https://gitee.com/api/v5/users/Spaceghost/followers",
                "following_url": "https://gitee.com/api/v5/users/Spaceghost/following{/other_user}",
                "gists_url": "https://gitee.com/api/v5/users/Spaceghost/gists{/gist_id}",
                "starred_url": "https://gitee.com/api/v5/users/Spaceghost/starred{/owner}{/repo}",
                "subscriptions_url": "https://gitee.com/api/v5/users/Spaceghost/subscriptions",
                "organizations_url": "https://gitee.com/api/v5/users/Spaceghost/orgs",
                "repos_url": "https://gitee.com/api/v5/users/Spaceghost/repos",
                "events_url": "https://gitee.com/api/v5/users/Spaceghost/events{/privacy}",
                "received_events_url": "https://gitee.com/api/v5/users/Spaceghost/received_events",
                "type": "User",
                "site_admin": false
            },
            "parents": [{
                "sha": "553c2077f0edc3d5dc5d17262f6aa498e69d6f8e",
                "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/commits/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e",
                "html_url": "https://github.com/tantao700/hello-world/commit/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e"
            }]
        },
        {
            "sha": "7fd1a60b01f91b314f59955a4e4d4e80d8edf11d",
            "node_id": "MDY6Q29tbWl0MTI5NjI2OTo3ZmQxYTYwYjAxZjkxYjMxNGY1OTk1NWE0ZTRkNGU4MGQ4ZWRmMTFk",
            "commit": {
                "author": {
                    "name": "The Octocat",
                    "email": "octocat@nowhere.com",
                    "date": "2012-03-06T23:06:50Z"
                },
                "committer": {
                    "name": "The Octocat",
                    "email": "octocat@nowhere.com",
                    "date": "2012-03-06T23:06:50Z"
                },
                "message": "Merge pull request #6 from Spaceghost/patch-1\n\nNew line at end of file.",
                "tree": {
                    "sha": "b4eecafa9be2f2006ce1b709d6857b07069b4608",
                    "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/git/trees/b4eecafa9be2f2006ce1b709d6857b07069b4608"
                },
                "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/git/commits/7fd1a60b01f91b314f59955a4e4d4e80d8edf11d",
                "comment_count": 61,
                "verification": {
                    "verified": false,
                    "reason": "unsigned",
                    "signature": null,
                    "payload": null
                }
            },
            "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/commits/7fd1a60b01f91b314f59955a4e4d4e80d8edf11d",
            "html_url": "https://github.com/tantao700/hello-world/commit/7fd1a60b01f91b314f59955a4e4d4e80d8edf11d",
            "comments_url": "https://gitee.com/api/v5/repos/tantao700/hello-world/commits/7fd1a60b01f91b314f59955a4e4d4e80d8edf11d/comments",
            "author": {
                "login": "octocat",
                "id": 583231,
                "node_id": "MDQ6VXNlcjU4MzIzMQ==",
                "avatar_url": "https://avatars3.githubusercontent.com/u/583231?v=4",
                "gravatar_id": "",
                "url": "https://gitee.com/api/v5/users/octocat",
                "html_url": "https://github.com/octocat",
                "followers_url": "https://gitee.com/api/v5/users/octocat/followers",
                "following_url": "https://gitee.com/api/v5/users/octocat/following{/other_user}",
                "gists_url": "https://gitee.com/api/v5/users/octocat/gists{/gist_id}",
                "starred_url": "https://gitee.com/api/v5/users/octocat/starred{/owner}{/repo}",
                "subscriptions_url": "https://gitee.com/api/v5/users/octocat/subscriptions",
                "organizations_url": "https://gitee.com/api/v5/users/octocat/orgs",
                "repos_url": "https://gitee.com/api/v5/users/octocat/repos",
                "events_url": "https://gitee.com/api/v5/users/octocat/events{/privacy}",
                "received_events_url": "https://gitee.com/api/v5/users/octocat/received_events",
                "type": "User",
                "site_admin": false
            },
            "committer": {
                "login": "octocat",
                "id": 583231,
                "node_id": "MDQ6VXNlcjU4MzIzMQ==",
                "avatar_url": "https://avatars3.githubusercontent.com/u/583231?v=4",
                "gravatar_id": "",
                "url": "https://gitee.com/api/v5/users/octocat",
                "html_url": "https://github.com/octocat",
                "followers_url": "https://gitee.com/api/v5/users/octocat/followers",
                "following_url": "https://gitee.com/api/v5/users/octocat/following{/other_user}",
                "gists_url": "https://gitee.com/api/v5/users/octocat/gists{/gist_id}",
                "starred_url": "https://gitee.com/api/v5/users/octocat/starred{/owner}{/repo}",
                "subscriptions_url": "https://gitee.com/api/v5/users/octocat/subscriptions",
                "organizations_url": "https://gitee.com/api/v5/users/octocat/orgs",
                "repos_url": "https://gitee.com/api/v5/users/octocat/repos",
                "events_url": "https://gitee.com/api/v5/users/octocat/events{/privacy}",
                "received_events_url": "https://gitee.com/api/v5/users/octocat/received_events",
                "type": "User",
                "site_admin": false
            },
            "parents": [{
                    "sha": "553c2077f0edc3d5dc5d17262f6aa498e69d6f8e",
                    "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/commits/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e",
                    "html_url": "https://github.com/tantao700/hello-world/commit/553c2077f0edc3d5dc5d17262f6aa498e69d6f8e"
                },
                {
                    "sha": "762941318ee16e59dabbacb1b4049eec22f0d303",
                    "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/commits/762941318ee16e59dabbacb1b4049eec22f0d303",
                    "html_url": "https://github.com/tantao700/hello-world/commit/762941318ee16e59dabbacb1b4049eec22f0d303"
                }
            ]
        }
    ],
    "files": [{
        "sha": "980a0d5f19a64b4b30a87d4206aade58726b60e3",
        "filename": "README",
        "status": "modified",
        "additions": 1,
        "deletions": 1,
        "changes": 2,
        "blob_url": "https://github.com/tantao700/hello-world/blob/7fd1a60b01f91b314f59955a4e4d4e80d8edf11d/README",
        "raw_url": "https://github.com/tantao700/hello-world/raw/7fd1a60b01f91b314f59955a4e4d4e80d8edf11d/README",
        "contents_url": "https://gitee.com/api/v5/repos/tantao700/hello-world/contents/README?ref=7fd1a60b01f91b314f59955a4e4d4e80d8edf11d",
        "patch": "@@ -1 +1 @@\n-Hello World!\n\\ No newline at end of file\n+Hello World!"
    }]
}