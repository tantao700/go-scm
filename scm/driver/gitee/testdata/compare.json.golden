[
    {
        "Path": "README",
        "Added": false,
        "Renamed": false,
        "Deleted": false
    }
]
